packages/cli/lib/oauth.js

```diff
@@ -95,7 +95,7 @@ const authenticateWithOauth = async configData => {
   const oauthManager = setupOauth(accountId, configData);
   logger.log('Authorizing');
   await authorize(oauthManager);
-  addOauthToAccountConfig(accountId, oauthManager);
```

Comment:
Should we still be doing the portalId parse on line 94?

Reply:
Good catch. Yeah we still need to do that. I moved it into `setupOauth` because that is where it's needed and it doesn't make sense to do the parsing in `authenticateWithOauth` anymore.